src/bp/ui-studio/src/web/views/OneFlow/diagram/SubworkflowForm/index.tsx

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios'
+import { FlowNode, FormField } from 'botpress/sdk'
+import { Contents, lang, MainContent, RightSidebar } from 'botpress/shared'
+import { FlowView } from 'common/typings'
+import React, { FC, useState } from 'react'
+
+import style from './style.scss'
+
+interface Props {
+  close: () => void
+  node: FlowNode
+  diagramEngine: any
+  flows: FlowView[]
+  type: 'in' | 'out'
+}
+
+const SubworkflowForm: FC<Props> = ({ type, close, node, diagramEngine, flows }) => {
```

Comment:
I know both in and out are similar enough but there is a lot of if(in) ... and logic is duplicated anyways .. can we split those 2 cases in 2 different components